packages/core/src/ctype/CType.spec.ts

```diff
@@ -135,11 +135,10 @@ describe('CType', () => {
       SDKErrors.ERROR_OBJECT_MALFORMED()
     )
     expect(() => CType.fromCType(invalidAddressCtype))
-      .toThrowErrorMatchingInlineSnapshot(`
-      "Provided DID identifier address invalid 
-
-          Address: Dp6K4tpdZtY3rNqM2uorQmsS6d3woxtnWMHjtzGftHmDb41N"
-    `)
+      // eslint-disable-next-line prettier/prettier
```

Comment:
Apparently using the prettier suggestions, the snapshots, which were identical, were failing to match for equality. So the `'` and the `"` have to be on the same line for it to succeed.